pg_party.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "activerecord", ">= 5.0", "< 6.1"
   spec.add_runtime_dependency "ruby2_keywords", "~> 0.0.2"
+  spec.add_runtime_dependency "parallel"
```

Comment:
Can we add some tighter constraints on this dependency? Like `~> 1.x` (unsure what minimum version is required)

Reply:
Set it to `~> 1.0` in https://github.com/webandtech/pg_party/commit/b798ce01da8005bd91df6a36e3265e6dd28cc7a7